apps/prairielearn/assets/scripts/institutionAdminAdminsClient.ts

```diff
@@ -0,0 +1,13 @@
+import { onDocumentReady, templateFromAttributes } from '@prairielearn/browser-utils';
+
+onDocumentReady(() => {
+  const removeAdminModal = document.getElementById('removeAdminModal');
+  document.querySelectorAll<HTMLButtonElement>('.js-remove-admin').forEach((el) => {
+    el.addEventListener('click', () => {
+      templateFromAttributes(el, removeAdminModal, {
+        'data-name-and-uid': '.js-name-and-uid',
+        'data-institution-administrator-id': '.js-institution-administrator-id',
+      });
+    });
+  });
```

Comment:
This is a little different from the pattern established in PT. There, we would listen for the `show.bs.modal` event on the modal and template it from `e.relatedTarget`. However, Bootstrap 4 doesn't use native events, so we'd have to use jQuery to listen for the event. In the interest of not introducing more jQuery code, I opted to flip it around: we attach a click listener to the button and template the modal when it's triggered.